atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -232,6 +235,28 @@ def get_registries(workflow, fallback=NO_FALLBACK):
     return registries_cm
 
 
+def get_docker_registry(workflow, fallback=NO_FALLBACK):
+    try:
+        all_registries = get_config(workflow).conf['registries']
+    except KeyError:
+        if fallback != NO_FALLBACK:
+            return fallback
+        raise
+
+    for registry in all_registries:
+        reguri = RegistryURI(registry.get('url'))
+        if reguri.version == 'v2':
+            regdict = {
+                'url': registry,
```

Comment:
Should this be `reguri`?

Reply:
Ah! It probably works as is because of RegistryURI.__repr__ method, which calls `.uri`. So this should probably be `registry.uri` to avoid confusion.

Reply:
Fixed.